libraries/abstractions/transport/secure_sockets/transport_secure_sockets.c

```diff
@@ -145,6 +149,10 @@ int32_t SecureSocketsTransport_Send( NetworkContext_t * pNetworkContext,
 
 /*-----------------------------------------------------------*/
 
+/* MISRA Rule 18.3 flags the following line for not using the const qualifier
+ * on `pNetworkContext`. Indeed, the object pointed by it is not modified
+ * by Secure Sockets, but other implementations of `TransportRecv_t` may do so. */
+/* coverity[misra_c_2012_rule_8_13_violation] */
```

Comment:
Same comment as the other PR, if it doesn't actually suppress the violations, we should leave it out